app/Commands/Extra/Compile/Options.hs

```diff
@@ -57,8 +57,7 @@ parseCompileOptions = do
       )
   _compileTerm <-
     switch
-      ( short 'T'
-          <> long "only-term"
+      ( long "only-term"
```

Comment:
Why remove the short '-T' version? I think it's handy and it fits with the other short upper-case options which modify the exact output format for other backend (-E, -S).